scripts/Tools/check_input_data

```diff
@@ -62,91 +57,6 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
     return args.svn_loc, args.input_data_root, args.data_list_dir, args.download
 
 ###############################################################################
```

Comment:
Moved most of this to a library so other tools can call check_input_data without having to fork a subprocess.